README.md

```diff
@@ -18,10 +18,3 @@ About this barclamp
 [![Build Status](https://travis-ci.org/crowbar/barclamp-hyperv-compute.svg?branch=master)](https://travis-ci.org/crowbar/barclamp-hyperv-compute)
 
 Information for this barclamp is maintained on the [Crowbar Framework Wiki](https://github.com/crowbar/crowbar/wiki)
-
-Legals
-------
-
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
-an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
-specific language governing permissions and limitations under the License.
```

Comment:
Heh, now you removed too much stuff :-) There was just a new line that got added ;-)